MyCryptoHQ/MyCrypto
# Remove REP under Equivalent Values

Closes https://github.com/MyCryptoHQ/MyCrypto/issues/2097

### Description

No longer includes Augur(REP) equivlent values, while viewing your wallet balance currency exchange.

### Changes

* Removed refrences to REP

### Steps to Test

1. Connect & unlock wallet
2. Check all paths under /account/
3. Verify that REP is no longer included with BTC and ETH. 

### Screenshots

Before: https://user-images.githubusercontent.com/26891718/43655628-424dc2c8-9704-11e8-9916-576d85c85954.png

After: https://i.imgur.com/umkXA6k.png
